src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -53,46 +53,77 @@ class ThroughputTypes(str, Enum):
 
     with self.argument_context('cosmosdb create') as c:
         c.argument('account_name', completer=None)
         c.argument('key_uri', help="The URI of the key vault", is_preview=True)
-        c.argument('enable_free_tier', arg_type=get_three_state_flag(), help="If enabled the account is free-tier.", is_preview=True)
-        c.argument('assign_identity', nargs='*', help="accept system or user assigned identities separated by spaces. Use '[system]' to refer system assigned identity. Currently only system assigned identity is supported.", is_preview=True)
+        c.argument('enable_free_tier', arg_type=get_three_state_flag(),
+                   help="If enabled the account is free-tier.", is_preview=True)
+        c.argument('assign_identity', nargs='*',
+                   help="accept system or user assigned identities separated by spaces. Use '[system]' to refer system assigned identity. Currently only system assigned identity is supported.", is_preview=True)
+        c.argument('is_restore_request', options_list=['--is-restore-request', '-r'], arg_type=get_three_state_flag(
+        ), help="Restore from an existing/deleted account.", is_preview=True, arg_group='Restore')
+        c.argument('restore_source', help="The restorable-database-account Id of the source account from which the account has to be restored. Required if --is-restore-request is set to true.", is_preview=True, arg_group='Restore')
+        c.argument('restore_timestamp', action=UtcDatetimeAction,
+                   help="The timestamp to which the account has to be restored to. Required if --is-restore-request is set to true.", is_preview=True, arg_group='Restore')
+        c.argument('databases_to_restore', nargs='+',
+                   action=CreateDatabaseRestoreResource, is_preview=True, arg_group='Restore')
 
     for scope in ['cosmosdb create', 'cosmosdb update']:
         with self.argument_context(scope) as c:
             c.ignore('resource_group_location')
             c.argument('locations', nargs='+', action=CreateLocation)
             c.argument('tags', arg_type=tags_type)
-            c.argument('default_consistency_level', arg_type=get_enum_type(DefaultConsistencyLevel), help="default consistency level of the Cosmos DB database account")
-            c.argument('max_staleness_prefix', type=int, help="when used with Bounded Staleness consistency, this value represents the number of stale requests tolerated. Accepted range for this value is 1 - 2,147,483,647")
+            c.argument('default_consistency_level', arg_type=get_enum_type(
+                DefaultConsistencyLevel), help="default consistency level of the Cosmos DB database account")
+            c.argument('max_staleness_prefix', type=int,
+                       help="when used with Bounded Staleness consistency, this value represents the number of stale requests tolerated. Accepted range for this value is 1 - 2,147,483,647")
```

Comment:
Can you add a validator for this parameter to check the value within allowed range?

Reply:
This change was reflected because of formatting issue which I fixed. I didn't introduce or modified this field as part of this PR.